### DISCUSSION TITLE
Security Onion 2.x on Red Hat 7/8/9???

### DISCUSSION THREAD

**Discussion by Mr2Ez** [Security-Onion-Solutions/securityonion#11339] [Unsupported Versions] [answered]
Hello all!

I know there is some animosity with a lot of folks on here regarding the Red Hat and CentOS debacle and I don't disagree, however, will Security Onion 2.x run on Red Hat as it does CentOS 7? My DoD organization will not allow us to use Rocky Linux due to compliance reasons, but we rely heavily on Secuirty Onion 2 for specific IT operations.

Any assitance would be appreciated....I'm very concerned that the directory structure between Red Hat and CentOS is not 1 to 1, but I'm not 100% sure. What are the directories that need modification for the setup wizard to run correctly?

Thanks in advance!!!!

**Comment by dougburks** [comment, marked as answer]
> My DoD organization will not allow us to use Rocky Linux due to compliance reasons, but we rely heavily on Secuirty Onion 2 for specific IT operations.

Our new Security Onion 2.4 ISO image is based on Oracle Linux 9 which is a RHEL 9 rebuild and accepted by most DoD organizations. If for some reason your organization does not accept Oracle Linux 9, then you may be able to do a manual installation on top of RHEL 9 but please note the support warnings:
https://docs.securityonion.net/en/2.4/os.html
https://blog.securityonion.net/2023/07/security-onion-24-base-os.html

**Reply by Mr2Ez** [reply]
Thanks sir, appreciated. Going to try RHEL 9...
